html/themes/custom/common_design_subtheme/templates/user/admin-menu.html.twig

```diff
@@ -0,0 +1,58 @@
+{{ attach_library('common_design_subtheme/rw-admin-menu') }}
+
+<section id="admin-menu" class="admin-menu">
+
+  {# Dropdown toggle is created with javascript #}
+
+  <h2 id="admin-menu-title">{% trans %}Admin menu{% endtrans %}</h2>
+
+  <div class="cd-global-header__dropdown cd-ocha-dropdown cd-dropdown" id="admin-menu-dropdown" aria-labelledby="admin-menu-title" data-cd-toggable="{{ 'Admin menu'|t }}" data-cd-component="admin-menu" data-cd-icon="arrow-down" data-cd-replace="admin-menu-title">
+
+    <div class="admin-menu-dropdown__inner">
+      <nav>
+        <ul>
+          <li>
+            <h3 id="admin-menu-create">Create</h3>
+            <ul aria-labelledby="admin-menu-create">
+              <li><a href="/node/add/report">Report</a></li>
+              <li><a href="/node/add/job">Job</a></li>
+              <li><a href="/node/add/training">Training</a></li>
+              <li><a href="/node/add/topics">Topic</a></li>
+              <li><a href="/admin/structure/taxonomy/country/add">Country</a></li>
+              <li><a href="/admin/structure/taxonomy/disaster/add">Disaster</a></li>
+              <li><a href="/admin/structure/taxonomy/source/add">Source</a></li>
```

Comment:
Looks like the urls to create terms have change in D8/9 to `admin/structure/taxonomy/manage/{vocabulary}/add`

```suggestion
              <li><a href="/admin/structure/taxonomy/manage/country/add">Country</a></li>
              <li><a href="/admin/structure/taxonomy/manage/disaster/add">Disaster</a></li>
              <li><a href="/admin/structure/taxonomy/manage/source/add">Source</a></li>
```